ports/espressif/common-hal/_bleio/Adapter.c

```diff
@@ -201,6 +221,7 @@ static int _scan_event(struct ble_gap_event *event, void *scan_results_in) {
             disc->data,
             disc->length_data);
     } else {
+        #if MYNEWT_VAL(BLE_EXT_ADV)
```

Comment:
What is the use case for not always having ` MYNEWT_VAL(BLE_EXT_ADV)` turned on?

Reply:
The ESP32 doesn't support BLE 5 so it doesn't support extended advertisements.